### PR TITLE
Add docs for helm enablement in kustomization TF provider

### PR DESCRIPTION
**Pull request by rattboi**

### CODE REVIEW

File: docs/data-sources/build.md [hunk 1]
```diff
@@ -25,6 +27,8 @@ data "kustomization_build" "test" {
 #### Child attributes
 
 - `load_restrictor` - setting this to `"none"` disables load restrictions
+- `enable_helm` - setting this to `true` enabled helm templating support
+- `helm_path` - set this to the path to find the helm executable (default: `"helmV3"`)
```
Comment (pst) [inline docs/data-sources/build.md:31]: The binary is called `helmV3` by upstream?

File: docs/data-sources/build.md [hunk 2]
```diff
@@ -25,6 +27,8 @@ data "kustomization_build" "test" {
 #### Child attributes
 
 - `load_restrictor` - setting this to `"none"` disables load restrictions
+- `enable_helm` - setting this to `true` enabled helm templating support
```
Comment (pst) [inline docs/data-sources/build.md:30]:
```suggestion
- `enable_helm` - setting this to `true` allows referencing helm charts in the kustomization.yaml
```

File: docs/data-sources/build.md (hunk 1 above)
Comment (pst) [reply docs/data-sources/build.md:31]:
```suggestion
- `helm_path` - set this to the path to the binary (defaults to: `helm`)
```

File: docs/data-sources/overlay.md [hunk 3]
```diff
@@ -269,13 +269,17 @@ data "kustomization_overlay" "example" {
 #### Child attributes
 
 - `load_restrictor` - setting this to `"none"` disables load restrictions
+- `enable_helm` - setting this to `true` enabled helm templating support
```
Comment (pst) [inline docs/data-sources/overlay.md:272]: Same as above

File: docs/data-sources/overlay.md [hunk 4]
```diff
@@ -269,13 +269,17 @@ data "kustomization_overlay" "example" {
...
+- `helm_path` - set this to the path to find the helm executable (default: `"helmV3"`)
```
Comment (pst) [inline docs/data-sources/overlay.md:273]: Same as above

**Review by pst — CHANGES_REQUESTED** [review]

File: docs/data-sources/build.md (hunk 1 above)
Comment (rattboi) [reply docs/data-sources/build.md:31]: it is indeed called `helmV3` by upstream. I'll correct the rest however.

**Comment by rattboi** [comment]
Realized I didn't leave proof for kustomize defaulting to `helmV3` as the binary. Here it is:

https://github.com/kubernetes-sigs/kustomize/blob/f8dad80a79f9d64776dfb27a3eb7c8d4b0835275/api/types/pluginconfig.go#L31

**Comment by pst** [comment]
Fucking hell, I already regret getting anywhere near helm with this...

**Pull request merged**